src/core/Akka.Tests/Actor/Stash/ActorWithBoundedStashSpec.cs

```diff
@@ -206,5 +215,34 @@ public void An_actor_with_stash_must_get_stash_capacity_from_aliased_dispatchers
             var stasher = Sys.ActorOf(Props.Create<StashingActorWithOverflow>().WithDispatcher("my-aliased-dispatcher-2"));
             TestStashOverflowException(stasher);
         }
+
+        [Fact]
+        public void An_actor_with_stash_must_get_stash_capacity_from_deployment()
+        {
+            // deployment configuration settings should override dispatcher settings
+            var stasher = Sys.ActorOf(Props.Create<StashingActorWithOverflow>().WithDispatcher("my-aliased-dispatcher-2"), "configStashingActor");
+            TestStashOverflowException(stasher, 2);
+        }
+        
+        /// <summary>
+        /// HOCON value should always beat the C# value
+        /// </summary>
+        [Fact]
+        public void An_actor_with_stash_must_get_stash_capacity_from_deployment_overridden_by_config()
```

Comment:
Validate that HOCON stash capacity always overrides the one set in C# (consistent with how every other setting works.)